jekyll/_cci2/authentication.adoc

```diff
@@ -5,48 +5,63 @@
 * Contact https://support.circleci.com[CircleCI support] and file a feature request for CircleCI Server.
 // still required?
 
 NOTE: After completing this configuration, all users must log in to CircleCI with their LDAP credentials. After logging in to CircleCI, each user will then click the Connect button on the Accounts page to connect and authenticate their GitHub account.
 
-== Configure LDAP Authentication
+=== Configure LDAP Authentication
+
+Below is an example configuration to give an idea of the information types required. The example shows OpenLDAP in use but settings will be comparable to what's required when using Active Directory:
+
+.LDAP Config Example
+image::LDAP_example.png[LDAP Example]
 
 This section provides the steps to configure LDAP in the CircleCI Server Management Console:
 
 . Verify access over the LDAP/AD ports to your LDAP/AD servers.
 . Log in as administrator to the Management Console for your newly installed CircleCI 2.0 instance.
 . Navigate to the Settings page and check the Enable LDAP-only Authentication button. Select either OpenLDAP or Active Directory.
-+
-.Enable LDAP
-image::LDAP.png[Enable LDAP]
 . Fill in your LDAP instance Hostname and port number.
+// how do they do this?
 . Select the encryption type (plain text is not recommended).
+// change screenshot
 . Fill in the Search user field with the LDAP admin username using the format `cn=<admin>,dc=<example>,dc=<org>` replacing `admin`, `example`, and `org` with appropriate values for your datacenter.
-. Fill in the Search password field with the LDAP admin password.
-. Fill in the User search DN field with an approrpiate value using the format `ou=<users>` replacing `users` with the value used in your LDAP instance.
-. Fill in the Username field with an approriate unique identifier used for your users, for example, `mail`.
+. Fill in the Search password field with your LDAP admin password.
+. Fill in the User search DN field with an appropriate value using the format `ou=<users>` replacing `users` with the value used in your LDAP instance.
+. Fill in the Username field with an appropriate unique identifier used for your users, for example, `mail`.
 . Fill in the Group Membership field with an appropriate value. By default, the value is `uniqueMember` for OpenLDAP and `member` for Active Directory. This field will list member `dn` for a group.
-. Fill in the Group Object Class field with an approrpiate value. By default, the value is `groupOfUniqueNames` for OpenLDAP and `group` for Active Directory. The value of the `objectClass` field indicates a `dn` is a group.
-. (Optional) Fill in the Test username and Test password fields with a test email and password for an LDAP user you want to test.
+. Fill in the Group Object Class field with an appropriate value. By default, the value is `groupOfUniqueNames` for OpenLDAP and `group` for Active Directory. The value of the `objectClass` field indicates a `dn` is a group.
```

Comment:
This seems almost right. I would edit it probably a bit: A name of an Object Class that will identify DN as a group. Example: `groupOfUniqueNames`

Reply:
Done